manual/core/custom_codecs/README.md

```diff
@@ -256,6 +256,18 @@ that maps instances of that class to Json strings, using a newly-allocated, defa
 It is also possible to pass a custom `ObjectMapper` instance using [ExtraTypeCodecs.json(Class,
 ObjectMapper)] instead.
 
+#### Mapping CQL vectors to Java array
+
+By default, the driver maps CQL `vector` to the [CqlVector] value type. If you prefer to deal with
+arrays, the driver offers the following codec:
+
+| Codec               | CQL type        | Java type |
+|---------------------|-----------------|-----------|
+| [ExtraTypeCodecs.floatVectorToArray(dimensions)] | `vector<float>` | `float[]` |
```

Comment:
As above, I should've added the Javadoc link below when I put this together originally.  That's what I get for writing docs instead of getting some sleep. :(

Link added.  I also changed the arg for floatVectorToArray() to be the type ("int") rather than the name ("dimensions") to match the usage elsewhere in this doc.